pyiron_atomistics/atomistics/master/quasi.py

```diff
@@ -173,3 +173,60 @@ def optimise_volume(self, bulk_eng):
             entropy_lst.append(entropy[ind])
             cv_lst.append(cv[ind])
         return v0_lst, free_eng_lst, entropy_lst, cv_lst
+
+    def plot_free_energy_volume_temperature(self,
+                                            murnaghan_job,
+                                            temperature_start=None,
+                                            temperature_end=None,
+                                            temperature_steps=None):
+        """
+        Plot volume vs free energy curves for defined temperatures.
+
+        Args:
+            murnaghan_job: job_name or job_id of the Murnaghan job
+            temperature_start: if None, value will be used from job.input['temperature_start']
+            temperature_end: if None, value will be used from job.input['temperature_end']
+            temperature_steps: if None, value will be used from job.input['temperature_steps']
+
+        Returns:
+            Volume vs free energy curves for defined temperatures.
+        """
+        job_murn = project.load(murnaghan_job)
+
+        if not self.status.finished:
+            raise ValueError(
+                "QHA Job must be successfully run, before calling this method."
+            )
+
+        if not job_murn.status.finished:
+            raise ValueError(
+                "Murnaghan Job must be successfully run, before calling this method."
+            )
+
+        import matplotlib
+        cmap = matplotlib.cm.get_cmap('coolwarm')
```

Comment:
Does it make sense to use a binary color map by default?  I'd rather suggest 'magma' or 'viridis'.  Best would probably be an extra argument to change the color map.